### PR TITLE
Added implementation for profile and .gitignore file

### PR DESCRIPTION
**Pull request by ALai2**
Implemented functions to get and post requests for profile. I also created a .gitignore file which ignores the target files when git searches for changed files. Target files have also been removed from tracking.

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/ProfileServlet.java [hunk 1]
```diff
@@ -96,7 +99,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       boolean glutenFree = Boolean.parseBoolean(request.getParameter("glutenFree"));
       boolean dairyFree = Boolean.parseBoolean(request.getParameter("dairyFree"));
       String[] allergies = request.getParameterValues("allergies");
-
+
```
Comment (joeychin01) [inline src/main/java/com/google/sps/servlets/ProfileServlet.java:102]: nit: is this change required?

Comment (ALai2) [reply src/main/java/com/google/sps/servlets/ProfileServlet.java:102]: The change from String[] to ArrayList<String> was necessary due to the way datastore returned the list and datastore does not store String[]. The type of id was changed from long to String because the String given by userService gave a numberFormatException when I tried to convert it to long.
